scripts/release.js

```diff
@@ -10,9 +10,11 @@ program
   .option('-v, --version <version>', 'specify version name to release, otherwise it will be generated randomly', '')
   .option('-s, --seed <seed>', 'specify seed to generate random version name', '')
   .option('-t, --token <token>', 'github access token', '')
+  .option('--no-dependency-check', 'do not check range when update dependencies')
```

Comment:
What's about `skip` instead of `no` to easily understand about what's going on?

Reply:
Use the same naming as other parameters. We can update it, but we should update the others :)

Reply:
I let you decide which one sounds more appropriate:)